src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api/Controllers/ApprenticeshipController.cs

```diff
@@ -171,5 +171,25 @@ public async Task<IActionResult> Resume([FromBody] ResumeApprenticeshipRequest r
             });
             return Ok(response);
         }
+
+        [HttpPost]
```

Comment:
should this be HttpPut?

The name of the method is `PostPut`???

Reply:
done